model/task/results.go

```diff
@@ -37,6 +37,10 @@ func (t *Task) ResultStatus() string {
 					status = evergreen.TaskSystemTimedOut
 				}
 			}
+
+		} else if t.Details.Type == "setup-failed" {
```

Comment:
Shouldn't "setup-failed" actually be `SetupCommandType`?

Reply:
done